packages/sit-onyx/src/components/OnyxRadioButton/OnyxRadioButton.vue

```diff
@@ -8,34 +9,128 @@ export type RadioButtonProps<TValue> = SelectionProps<TValue> & {
    * See also: https://html.spec.whatwg.org/multipage/input.html#radio-button-group
    */
   name: string;
+  isRequired?: boolean;
+  errorMessage?: string;
 };
 
 const props = defineProps<RadioButtonProps<TValue>>();
+
+const selector = ref<HTMLInputElement>();
```

Comment:
```suggestion
const selector = ref<HTMLInputElement | null>(null);
```

Template refs can be null if not mounted yet or due to "v-if". We should type it accordingly :)

Reply:
Also is selector a standard name? Imho something like "inputRef" would be clearer to me

Reply:
@BoppLi brought up the argument, that BEM should be semantical, not technical. To keep the naming in sync I also named the ref after this. I couldn't think of a better name

Reply:
@larsrickert Actually `null` is incorrect here. The initial value is `undefined` and by not passing one into to the `ref()` it already types it as `HTMLInputElement | undefined`. But I made the same error :D

Reply:
See [Vue docs](https://vuejs.org/guide/essentials/template-refs#accessing-the-refs).
See the following example where the ref will be `null`:

```vue
<script lang="ts" setup>
const inputRef = ref<HTMLInputElement | null>(null);

onMounted(() => {
  console.log(inputRef.value);
});
</script>

<template>
    <input v-if="false" />
</template>

```